analyses/cell-contamination-removal-analysis/run-cell-contamination-removal-analysis-steps-4.R

```diff
@@ -0,0 +1,50 @@
+#################################################################################
+# This will run all scripts in the module
+#################################################################################
+#################################################################################
+# Load library
+suppressPackageStartupMessages({
+  library(yaml)
+  library(tidyverse)
+  })
+
+#################################################################################
+# load config file
+configFile <- paste0("../../project_parameters.Config.yaml")
+if (!file.exists(configFile)){
+  cat("\n Error: configuration file not found:", configFile)
+  stop("Exit...")}
+
+# read `yaml` file defining the `params` of the project and strategy analysis
+yaml <- read_yaml(configFile)
+
+#################################################################################
+# Set up directories and paths to root_dir and analysis_dir
+root_dir <- yaml$root_dir
+analysis_dir <- file.path(root_dir, "analyses", "cell-contamination-removal-analysis") 
+report_dir <- file.path(analysis_dir, "plots") 
+
+################################################################################################################
+# step 4 - Run markers
+future_globals_value = yaml$future_globals_value_clustering_module
+resolution = yaml$resolution_find_markers
+
+rmarkdown::render('02-find-markers.Rmd', clean = TRUE,
```

Comment:
I believe the `Rmd` script called here should be `04-find-markers.Rmd` to be consistent with file structure from README and numbering of `Rmd` scripts called in `steps-1-2-3`.

```suggestion
rmarkdown::render('04-find-markers.Rmd', clean = TRUE,
```